### DISCUSSION TITLE
What are the undefinable characters for?

### DISCUSSION THREAD

**Discussion by erlendnagel** [paulscottrobson/neo6502-firmware#280] [Q&A] [answered]
I noted that characters 128 and above are blank. But only characters 192 - 255 appear to be redefinable via the API. I haven't been able to figure out why that is. Does anyone know?

**Comment by paulscottrobson** [comment, marked as answer]
128..143 and 144..159 set background and foreground colour. (bad decision but it is what it is). The others are reserved as are some of the 0..31 range)
